executor/cook/config.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python3
 
 import logging
+import os
 
 from pymesos.utils import parse_duration
 
+EXECUTOR_PROGRESS_OUTPUT_FILE = 'EXECUTOR_PROGRESS_OUTPUT_FILE'
```

Comment:
Maybe `DEFAULT_PROGRESS_FILE_ENV_VARIABLE` for the name of the constant? "executor" and "output" seem superfluous to me, while it might be nice to clarify that this is the default and that it's the name of an env variable

Reply:
Renamed to `DEFAULT_PROGRESS_FILE_ENV_VARIABLE `